test/topology_custom/test_change_replication_factor_1_to_0_with_tablets.py

```diff
@@ -0,0 +1,66 @@
+#
+# Copyright (C) 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+import logging
+import pytest
+import asyncio
+import time
+
+from cassandra import ConsistencyLevel  # type: ignore
+from cassandra.query import SimpleStatement  # type: ignore
+from test.pylib.manager_client import ManagerClient
+from test.pylib.util import wait_for_cql_and_get_hosts
+
+
+logger = logging.getLogger(__name__)
+CONFIG = {"endpoint_snitch": "GossipingPropertyFileSnitch", "enable_tablets": True}
+
+
+@pytest.mark.asyncio
+@pytest.mark.xfail(reason="issue #20282")
+async def test_change_replication_factor_1_to_0_with_tablets(request: pytest.FixtureRequest, manager: ManagerClient) -> None:
+    cmdline = [
+        # '--logger-log-level', 'storage_proxy=trace'
+    ]
+    logger.info("Creating a new cluster")
+    for i in range(2):
+        await manager.servers_add(
+            1,
```

Comment:
why using `servers_add(1, ...)` instead of just `server_add(...)` for a single server?

Reply:
done